### PR TITLE
patron: improve patron profile

### PR DESCRIPTION
**Pull request by zannkukai**
* Refactoring the public patron profile for a better responsive display.
* Closes rero/rero-ils#2751.

Co-authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## How to test?

image

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: projects/public-search/src/app/patron-profile/patron-profile-fees/patron-profile-fee/patron-profile-fee.component.html [hunk 1]
```diff
@@ -15,112 +15,94 @@
...
+              <dt class="col-2 label-title" translate>Title</dt>
+              <dd class="col-10">
+                <a href="/{{ organisation.code }}/documents/{{ document.pid }}">{{ document.title | mainTitle }}</a>
+              </dd>
+              <!-- CALL NUMBER -->
+              <ng-container *ngIf="record.metadata.loan.item.call_number">
+                <dt class="col-2 label-title" translate>Call number</dt>
+                <dd class="col-10">{{ record.metadata.loan.item.call_number }}</dd>
+              </ng-container>
+              <!-- TITLE -->
```
Comment (Garfield-fr) [inline projects/public-search/src/app/patron-profile/patron-profile-fees/patron-profile-fee/patron-profile-fee.component.html:52]: The text is wrong

File: projects/public-search/src/app/patron-profile/patron-profile-histories/patron-profile-histories.component.ts [hunk 2]
```diff
@@ -84,6 +84,7 @@ export class PatronProfileHistoriesComponent implements OnInit, OnDestroy {
             this._historyQuery(1).subscribe((response: Record) => {
               this._paginator.setRecordsCount(response.hits.total.value);
               this.records = response.hits.hits;
+              console.log(this.records);
```
Comment (Garfield-fr) [inline projects/public-search/src/app/patron-profile/patron-profile-histories/patron-profile-histories.component.ts:87]: Remove this

**Review by Garfield-fr — APPROVED** [review]

File: projects/public-search/src/app/patron-profile/patron-profile-loans/patron-profile-loan/patron-profile-loan.component.html [hunk 3]
```diff
@@ -14,79 +14,63 @@
...
+          <span [ngClass]="{ 'font-weight-bold text-success': actionSuccess}">
+            {{ record.metadata.end_date | dateTranslate :'shortDate' }}
+          </span>
+        </div>
+        <div *ngIf="record.metadata.extension_count && record.metadata.extension_count > 0" class="badge badge-secondary font-weight-normal">
+          {{ record.metadata.extension_count }}
+          {{ record.metadata.extension_count | i18nPlural: {'=1': 'renewal', 'other': 'renewals'} | translate }}
+        </div>
+        <div *ngIf="record.metadata.is_late" class="badge badge-warning font-weight-normal" translate>overdue</div>
+        <div *ngIf="isDueSoon" class="badge badge-secondary font-weight-normal" translate>approching due date</div>
```
Comment (PascalRepond) [inline projects/public-search/src/app/patron-profile/patron-profile-loans/patron-profile-loan/patron-profile-loan.component.html:51]: appr**oa**ching due date

Mais je remplaçerais par "due soon" pour simplifier

**Comment by PascalRepond** [comment]
## PO tests

- [ ] REP: (not-blocking but nice-to-have) could we implement the same `expand all` button as in the pro UI? (see https://github.com/rero/rero-ils-ui/pull/844)

**Review by PascalRepond — APPROVED** [review]

**Review by benerken — APPROVED** [review]

**Pull request merged**